deploy-board/deploy_board/templates/configs/env_config.tmpl

```diff
@@ -7,13 +7,16 @@
                 <div class="form-group">
                     <label for="maxParallelHosts" class="deployToolTip control-label col-xs-2"
                         data-toggle="tooltip"
-                        title="Maximum number of hosts allowed to deploy at the same time">
-                        Maximum Hosts
+                        title="Maximum number (or percentage) of hosts allowed to deploy at the same time">
+                        Maximum Deploys In Parallel
```

Comment:
Deploy has been accepted as a higher level concept, usually they are in the context of environments. "Deploys in parallel" sounds like there are multiple deploys going on for a service.